pootle/apps/accounts/adapter.py

```diff
@@ -47,3 +50,13 @@ def is_open_for_signup(self, request):
     def add_message(self, request, level, message_template, *args, **kwargs):
         """Silence messages altogether."""
         pass
+
+    def send_confirmation_mail(self, *args, **kwargs):
+        try:
+            super(PootleAccountAdapter, self).send_confirmation_mail(*args,
+                                                                     **kwargs)
+        except Exception:
+            log("\n\n\nSign up failed. Couldn't sent confirmation email.\n\n")
```

Comment:
do you not want to do logger.error(e.message) - that way the sysadmin can see the exact failure

That is why i asked for it - because we swallow the raised exception and raise a different one - i think the original exception should be logged